js/source/worker_tile.js

```diff
@@ -226,7 +226,8 @@ WorkerTile.prototype.parse = function(data, layers, actor, callback) {
         callback(null, {
             elementGroups: elementGroups,
             buffers: buffers,
-            extent: extent
+            extent: extent,
+            bucketStats: self.bucketStats
```

Comment:
Is everyone ok with the perf impact of transferring this? Should be pretty small.